apps/console/src/features/core/components/header.tsx

```diff
@@ -99,22 +98,12 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
     const config: ConfigReducerStateInterface = useSelector(
         (state: AppState) => state.config
     );
-    const showAppSwitchButton: boolean = useSelector(
```

Comment:
If the code is removed altogether, myaccount menu item will not be available in the cloud offering as well.
If the myaccount menu item needs to be hidden only in onprem apps, let's turn it off from the deployment config.

Reply:
Ack, will check this.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4603/commits/8a78ad2cf6413b0f5a2e41dc02b10b6d1d6ea430